mne/annotations.py

```diff
@@ -0,0 +1,79 @@
+# Authors: Jaakko Leppakangas <jaeilepp@student.jyu.fi>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+from datetime import datetime
+import time
+import json
+
+
+class Annotations(object):
+    """Annotation object for annotating segments of raw data.
+
+    Parameters
+    ----------
+    onset: array of float, shape (n_annotations,)
+        Annotation time onsets from the beginning of the recording.
```

Comment:
You should probably mention somewhere that all values will eventually be sorted by onset time, instead of preserving their input order. BTW what's the reason behind sorting them?